app.json

```diff
@@ -1,12 +1,56 @@
 {
   "name": "viking-garage-frontend",
+  "logo": "https://res.cloudinary.com/hkhuw4b7v/image/upload/v1499880454/raido_logo_small_kx9txb.png",
+  "description": "A worldwide marketplace and community for real people riding real motorcycles. Local private rentals, repairs, tours. VG brings the Platform and logistics.",
+  "keywords": [
+    "motorcycle",
+    "rental",
+    "repair",
+    "tour",
+    "marketplace",
+    "community",
+    "global",
+    "local",
+    "private",
+    "worldwide",
+    "motocross",
+    "enduro",
+    "cafe racer",
+    "scrambler",
+    "chopper",
+    "bobber",
+    "brat style",
+    "adventure",
+    "braap"
+  ],
+  "website": "https://vikinggarage.com/",
+  "repository": "https://github.com/michalmikolajczyk/viking-garage-frontend",
   "scripts": {},
   "env": {
-    "API_URL": "https://viking-garage-api-staging.herokuapp.com/",
-    "NODE_MODULES_CACHE": "false",
-    "NPM_CONFIG_PRODUCTION": "false"
+    "API_URL": {
+      "description": "URL to API for server side render requests",
+      "value": "https://viking-garage-api-staging.herokuapp.com/",
+      "required": true
+    },
+    "NODE_ENV": {
+      "description": "Mode of running application",
+      "value": "production",
+      "required": true
+    },
+    "NODE_MODULES_CACHE": {
+      "description": "Heroku maintains a cache directory that is persisted between builds",
+      "value": false,
+      "required": true
+    },
+    "PORT": {
+      "description": "Port to run application server",
+      "value": 3000,
```

Comment:
this is most likely wrong. We cannot bind then app to any specific port on Heroku.